stdlib/public/core/Process.swift

```diff
@@ -85,3 +99,10 @@ func _stdlib_didEnterMain(
   Process._argc = CInt(argc)
   Process._unsafeArgv = argv
 }
+
+/// A pointer-sized value for atomics.
```

Comment:
It is not just useful for atomics, we are going to use it for other things (e.g., caches in value types).